Ginger/Ginger/Actions/ActionConversion/ConversionConfigurationWzardPage.xaml.cs

```diff
@@ -99,7 +99,7 @@ private bool IsPOMSupportedFromSelectedTargetApplications()
             foreach (string targetapp in TargetAppList)
             {                
                 ePlatformType platform = (from x in WorkSpace.Instance.Solution.ApplicationPlatforms where x.AppName == targetapp select x.Platform).FirstOrDefault();
-                if (platform != ePlatformType.NA)
+                if (platform != ePlatformType.NA && PlatformInfoBase.GetPlatformImpl(platform) != null)
```

Comment:
do we need to check in this ? because you mentioned Mahesh if working on Generic fix for this